invesalius/constants.py

```diff
@@ -858,3 +858,7 @@
 ROBOT_VERSOR_SCALE_FACTOR = 70
 #Robot Working Space is defined as 800mm in Elfin manual. For safety, the value is reduced by 5%.
 ROBOT_WORKING_SPACE = 760 #mm
+ROBOT_MOVE_STATE = {"free to move": 0,
```

Comment:
I guess enums (see https://docs.python.org/3/library/enum.html) would be a good way to implement these mappings from strings to constant values.

However, I'm not sure if it fits the current style in constants.py - for one thing, it would then be RobotMoveState class, instead of the constant ROBOT_MOVE_STATE.

Maybe we can consider changing this sometime later. For now, how you did it is probably fine.